keyform/forms.py

```diff
@@ -15,7 +15,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super(CreateForm, self).__init__(*args, **kwargs)
         # removes blank choices from Radio Select options
-        self.fields['payment_method'] = TypedChoiceField(widget=RadioSelect(), choices=Request.PAYMENT_TYPES)
+        self.fields['payment_method'] = TypedChoiceField(widget=RadioSelect(), choices=[('', _('Not Applicable'))] + list(Request.PAYMENT_TYPES), required=False)

```

Comment:
Because I wanted it to store as NULL for some reason... I don't remember that reason, though...

If you really want to change it, I'm open to that